### PR TITLE
Feature/book details view krd

### LINKED ISSUE #27: Add functionality to single page view

### PR DESCRIPTION
**Pull request by krdurrett**
## What Features are Being Added?
- Added a package called title case to normalize the book titles
- Added styling to the book details view.

## What is Being Refactored?
- Refactored api call for getting a single book to isbn, and changed appropriate routes to reflect isbn rather than id.

## Where should the reviewer start?
-Utils.js and then book detail css file.

## How should this be manually tested?
-n/a

## What are the relevant tickets?
-n/a

## Any Other Notes/Questions/Problems?
-n/a

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.3.0",
     "react-scripts": "5.0.0",
+    "title-case": "^3.0.3",
```
Comment (cagallo) [inline package.json:13]: This is such a cool npm package!

File: src/styles/BookDetails.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
+.book-details {
```
Comment (Fordo29) [inline src/styles/BookDetails.css:1]: Thanks for styling this Kayla

File: src/styles/BookDetails.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.book-details-text-container {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-between;
```
Comment (Fordo29) [inline src/styles/BookDetails.css:15]: This was a great learning for me

File: src/styles/BookDetails.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  font-size: x-large;
+}
+
+.selected-title {
+  font-size: xxx-large;
+  margin-top: 0;
+}
+
+.selected-cover {
+  box-shadow: 6px 9px 10px #595045, 12px 15px 10px #938673;
```
Comment (cagallo) [inline src/styles/BookDetails.css:33]: Really nice touch here!

File: src/utils.js [hunk 5]
```diff
@@ -1,12 +1,16 @@
+import { titleCase } from "title-case";
```
Comment (Fordo29) [inline src/utils.js:1]: Neat npm package to add

File: src/components/App.js [hunk 6]
```diff
@@ -38,8 +38,8 @@ class App extends Component {
         <Route exact path='/favorites' render={() => {
           return <Favorites />
         }} />
-        <Route exact path='/:id' render={({ match }) => {
-          return <BookDetails id={match.params.id} />
+        <Route exact path='/:isbn' render={({ match }) => {
+          return <BookDetails isbn={match.params.isbn} />
```
Comment (ectweitmann) [inline src/components/App.js:42]: Nice catch changing this to isbn!!

**Review by cagallo — APPROVED** [review]

File: src/utils.js [hunk 7]
```diff
@@ -1,12 +1,16 @@
...
+
 export const cleanBookData = (book) => {
+  const title = titleCase(book.title.toLowerCase());
```
Comment (ectweitmann) [inline src/utils.js:4]: This is so awesome. Nice find y'all!

**Review by ectweitmann — APPROVED** [review]

**Pull request merged**